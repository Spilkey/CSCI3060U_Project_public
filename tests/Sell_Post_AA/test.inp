loginuser01sellairpods meetup100.0026buyairpods meetup4user01logout
exit