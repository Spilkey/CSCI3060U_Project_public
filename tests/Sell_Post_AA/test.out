03_airpods_meetup____________user01________026_100.00
04_airpods_meetup____________user01________010_100.00
00_user01__________AA_000000.00